calculator.js

```diff
@@ -1,5 +1,49 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
-}
+// This line "loads" the prompt package and puts it into a variable we can use called prompt
+// Lines like this (that use require()) typically go on the top of the file
+const prompt = require("prompt");
+
+const calculateUserInput = function(error, promptInput) {
+    calculate(promptInput.num1, promptInput.num2, promptInput.operation);
+};
+
+const calculate = function(num1, num2, operation) {
+    switch (operation) {
+        case "a":
+        case "add":
+        case "addition":
+        case "+":
+            console.log(num1 + num2);
+            break;
+        case "s":
+        case "sub":
+        case "subtract":
+        case "-":
+            console.log(num1 - num2);
+            break;
+        case "d":
+        case "div":
+        case "divide":
+        case "/":
+            console.log(num1 / num2);
+            if (num2 == 0) {
+                console.log("Error");
+            }
+            break;
```

Comment:
You're trying to catch divide-by-zero errors, but you have the if statement **after** the console.log.

```suggestion
            if (num2 == 0) {
                console.log("Error");
            } else {
              console.log(num1 / num2);
            }
            break;
```